src/presetsController.ts

```diff
@@ -420,7 +429,7 @@ export class PresetsController {
 
             if (newPreset) {
 
-                const before: preset.ConfigurePreset[] = await this.getAllConfigurePresets();
+                const before: preset.ConfigurePreset[] = preset.allConfigurePresets(this.folderPath);
```

Comment:
Shouldn't this include the user presets?

Reply:
What do you mean? `allConfigurePresets()` also gets the user presets. 

Although, a related note is that `getAllConfigurePresets()` and `allConfigurePresets()` do exactly the same thing now that I took out expandVendor and expandConditions, except the former is in `PresetsController` and the latter is in `Preset`.

I wanted to get rid of `allConfigurePresets()` - but it has this comment "Don't use this function if you need to keep any changes in the presets" and I'm not sure what that is referring to or if it's just outdated.

Reply:
Oh I see. One is async and the other is not

Reply:
Ah, I see, yeah it would be great to only have one of those methods to avoid confusion. Are there any differences besides async? Are they using the same underlying code?

Reply:
Nope they're exactly the same except for async. I'm thinking I'll just keep the async one..?

Reply:
Will handle this in a follow up cleanup/refactor PR